lib/css/components/post-summary.less

```diff
@@ -143,12 +143,14 @@
     color: var(--fc-light);
 
     .s-post-summary--stats-item {
-        display: inline-flex;
-        gap: 0.3em;
-        align-items: center;
-        justify-content: center;
-        white-space: nowrap;
-        border: 1px solid transparent;
+        &:not(.s-badge) {
```

Comment:
optional: Other than `border`, we can get away with not excluding badge from any of these. Up to you on whether we want to do so or not.

Reply:
My preference is to apply as few styles to a nested component as possible.